api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1131,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: Subscription is a high level consumption unit.
+    properties:
+      application_id:
+        type: string
+        example: 'price-service'
+        description: The id of application that will consume from the subscription.
+      event_types:
```

Comment:
I don't think it will be possible to switch at later stage because it will be backwards incompatible change in the API - which is forbidden http://zalando.github.io/restful-api-guidelines/compatibility/Compatibility.html#must-dont-break-backward-compatibility
In the past we had such a change for event-pushing endpoint (from single event to array of events) and it caused a lot of pain to our users, and we were not even live.
That's why I think it will makes sense to make it an array from the very beginning because we will need to support that anyway very soon.